parser/src/app.ts

```diff
@@ -8,6 +8,8 @@ import { isUrlBlocked } from './isUrlBlocked';
 import { writeCsv, writeJson } from './writeFile';
 import { AzurePortal, getPrice, getPricing } from './azurePortalExtensions';
 
+let outputPath: string | undefined;
```

Comment:
issue: we need a safer way to assign a default to `outputPath`. If I run `crawl` without the output path parameter, the command fails. This is why the build failed on your PR and I can reproduce it locally:

```powerhsell
yarn crawl --culture en-us --currency usd --operating-system linux --region us-west
```
Notice the `undefined` in the path:

```plaintext
[Error: ENOENT: no such file or directory, open 'E:\code\me\azure-vm-pricing\parser\undefined\vm-pricing_us-west_linux.csv'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'E:\\code\\me\\azure-vm-pricing\\parser\\undefined\\vm-pricing_us-west_linux.csv'
}
```

Reply:
Variable definition modified so that the variable also has a default value. I really have no idea if this is correct, but it works.

At the same time, in the `case` where the input parameters are parsed, there is a condition that also watches that the variable must not be empty and if it is, the default value is set.

At the same time, it watches that the defined/default output path exists and if not, the parser creates it.